source/test/sgp_mode_test/CPUState_Test.test.cc

```diff
@@ -0,0 +1,54 @@
+#include "../../Empirical/include/emp/Evolve/World.hpp"
+#include "../../Empirical/include/emp/data/DataFile.hpp"
+#include "../../Empirical/include/emp/Evolve/Systematics.hpp"
+#include "../../Empirical/include/emp/math/random_utils.hpp"
+#include "../../Empirical/include/emp/math/Random.hpp"
+#include "../../Organism.h"
+#include "../../sgp_mode/CPU.h"
+#include "../../sgp_mode/SGPHost.h"
+#include "../../sgp_mode/SGPSymbiont.h"
+#include "../../sgp_mode/SGPWorld.h"
+#include "../../sgp_mode/Tasks.h"
+#include "../../sgp_mode/SymbiontImpact.h"
+#include "../../native/symbulation_sgp.cc"
+#include <set>
+#include <math.h>
+#include <tuple>
+
+TEST_CASE("Host Symbiont Test Chamber", "[sgp]"){
```

Comment:
Is this a unit test for CheckSymbiont? If so, it should be compared to the SymbiontImpact.test.cc file and if it's testing something different, moved there